### ISSUE TITLE
set_ylabel fail in triangular.py

### ISSUE THREAD

**Issue by Ebert-Daniel** [pycalphad/pycalphad#303]
I got a problem trying to plot the example "triangular projection". 

I got an error message:

triangular.py", line 200, in set_ylabel
    return super().set_ylabel(ylabel, fontdict, labelpad, loc=loc, **kwargs)

AttributeError: 'Text' object has no property 'loc'

it happens when I tried to se ylabel during the example tutorial.

Thank you
EA

**Comment by bocklund** [comment]
@Ebert-Daniel Thanks for reporting this, it looks like it was introduced in the last commit (be40446, #295). 

I can reproduce with 

```python
# pycalphad-303.py
import pycalphad
import matplotlib.pyplot as plt
plt.gca(projection='triangular')
plt.ylabel('label')
```

which raises

```python
Traceback (most recent call last):
  File "pycalphad-303.py", line 5, in <module>
    plt.ylabel('label')
  File "/Users/brandon/anaconda3/lib/python3.7/site-packages/matplotlib/pyplot.py", line 3072, in ylabel
    ylabel, fontdict=fontdict, labelpad=labelpad, **kwargs)
  File "/Users/brandon/Projects/pycalphad/pycalphad/plot/triangular.py", line 200, in set_ylabel
    return super().set_ylabel(ylabel, fontdict, labelpad, loc=loc, **kwargs)
  File "/Users/brandon/anaconda3/lib/python3.7/site-packages/matplotlib/axes/_axes.py", line 298, in set_ylabel
    return self.yaxis.set_label_text(ylabel, fontdict, **kwargs)
  File "/Users/brandon/anaconda3/lib/python3.7/site-packages/matplotlib/axis.py", line 1567, in set_label_text
    self.label.update(kwargs)
  File "/Users/brandon/anaconda3/lib/python3.7/site-packages/matplotlib/text.py", line 187, in update
    super().update(kwargs)
  File "/Users/brandon/anaconda3/lib/python3.7/site-packages/matplotlib/artist.py", line 916, in update
    ret = [_update_property(self, k, v) for k, v in props.items()]
  File "/Users/brandon/anaconda3/lib/python3.7/site-packages/matplotlib/artist.py", line 916, in <listcomp>
    ret = [_update_property(self, k, v) for k, v in props.items()]
  File "/Users/brandon/anaconda3/lib/python3.7/site-packages/matplotlib/artist.py", line 912, in _update_property
    raise AttributeError('Unknown property %s' % k)
AttributeError: Unknown property loc
```

Since you have a development environment working already, you should just be able to workaround this by checking out the last commit (`git checkout de1cebf`). If you'd be interested in contributing a fix and opening a pull request, we would be happy to help guide you through it.

**Comment by bocklund** [comment]
The above exception was trigged with matplotlib version 3.0.3, but the same code snippet works with matplotlib 3.3.3. 

It looks like matplotlib introduced the `loc` keyword argument to `set_ylabel` in https://github.com/matplotlib/matplotlib/pull/15974 and it was released in matplotlib version 3.3.0 (see the [release notes](https://matplotlib.org/stable/users/whats_new.html#align-labels-to-axes-edges)).

@Ebert-Daniel, it looks like updating your matplotlib to 3.3.0 or later should fix it.

A fix for this issue is then to either

1. Update our matplotlib version requirement to 3.3.0 or later
2. Stop using the newer matplotlib API option for `loc`. In our class, it's just passed through, so it's not necessary to have right now.


cc @richardotis
